tests/test_utils/test_multi_objective.py

```diff
@@ -10,61 +12,34 @@
 
 class MultiObjectiveTest(unittest.TestCase):
     def setUp(self):
-        self.bounds_1d = [(0, 1)]
-        self.bounds_2d = [(0, 1), (50, 100)]
+        self.bounds = [(0, 50), (50, 100)]
+        self.bounds_invalid = [(0, 0), (5, 5)]
 
     def test_normalize_costs(self):
-        # Normalize between 0..1 given data only
-        v = np.array([[5, 2], [10, 0]])
+        # If no bounds are passed, we get ones back
```

Comment:
you will have to change this here as well if norm no longer returns ones for bounds=none